packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -0,0 +1,131 @@
+import asyncStates from '@zooniverse/async-states'
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import * as d3 from 'd3'
+import exampleData from './example-planet-hunters-data.json'
+
+import LightCurveViewer from './LightCurveViewer'
+import locationValidator from '../../helpers/locationValidator'
+
+class LightCurveViewerContainer extends React.Component {
+  constructor () {
+    super()
+    this.state = {}
+    
+    this.svgContainer = React.createRef();
+    
+    //TODO: turn into variables?
+    this.width = 500
+    this.height = 500
+    
+    this.d3svg = null;
+    this.d3dataLayer = null;
+  }
+
+  componentDidMount () {
+    //Prepare the main SVG
+    //--------------------------------
+    this.d3svg = d3.select(this.svgContainer.current).append('svg')
+      .attr('class', 'light-curve-viewer')
+      //.attr('viewBox', `0 0 ${this.width} ${this.height}`)  //TODO: check
+      //.attr('style', 'width: 100%; height: 100%')  //TODO: check
+      .attr('width', this.width)
+      .attr('height', this.height)
+    
+    //Note: an alternative way to later select this node is
+    //`d3.select(this.d3svg.node())`. Don't use `d3.select(this.d3svg)`.
+    
+    //Deco: Background layer
+    this.d3svg.append('rect')
+      .attr('width', this.width)
+      .attr('height', this.height)
+      .attr('fill', '#f8f8f8')
+      .attr('stroke', '#333')
+      .attr('stroke-width', '2')
+    //--------------------------------
+    
+    //Prepare the data layer
+    //--------------------------------
+    this.d3dataLayer = this.d3svg.append('g')
+      .attr('class', 'data-layer')
+      .attr('width', this.width)
```

Comment:
SVG `g` elements are more like logical containers, and don't have dimensions

Reply:
DERP! Sorry, that was a poor copy-paste from my 'rect' code, I'll get rid of those dimensions.